### ISSUE TITLE
BuildShip support not working with BuildShip 2.0 on STS 3.9.0

### ISSUE THREAD

**Issue by justride** [spring-projects/spring-ide#169]
Unable to create a New Spring Starter Project using Gradle Buildship 2.x
"Can not import using Gradle (Buildship 2.x) because Buildship Gradle Tooling version 2.x is not installed. You can install it from Eclipse Marketplace."

STS and Buildship are both latest release:

Spring Tool Suite 
Version: 3.9.0.RELEASE
Build Id: 201707061903
Platform: Eclipse Neon.3 (4.6.3)

  Buildship: Eclipse Plug-ins for Gradle	2.1.1.v20170713-0942	org.eclipse.buildship.feature.group	Eclipse Buildship

**Comment by dejan2609** [comment]
Similar problem: I'm also unable to create New Spring Starter project:   _**Type**_ drop-down list is missing _**BuildShip 2.0**_ entry
```
Spring Tool Suite 
Version: 3.9.0.RELEASE
Build Id: 201707061823
Platform: Eclipse Oxygen (4.7.0)
OS: Windows 10 64 bit
Buildship: 2.1.1.v20170713-0944	
```
![image](https://user-images.githubusercontent.com/19467899/28239690-5ac3dd8a-6972-11e7-983b-07e9670597fa.png)

**Comment by dejan2609** [comment]
Seems like a regression of #90 _**(BuildShip support not working with BuildShip 2.0)**_

**Comment by dejan2609** [comment]
Correction: actually, this is a duplicate of  #163 **_(Spring Tool Suite 3.9 not working with Gradle Buildship 2.1.x_)**

**Comment by martinlippert** [comment]
closing this one as a duplicate of #163

**Comment by martinlippert** [comment]
Please also notice that there is a workaround described in +163 (which means to basically install Buildship 2.0.x instead of 2.1.x)

**Comment by martinlippert** [comment]
Buildship 2.1.2 got released and works nicely again with STS 3.9.0.
